core/templates/timeline/_timeline.html

```diff
@@ -17,48 +17,52 @@ <h3 class="text-center">
 </h3>
 {% if timeline_objects %}
     <ul class="timeline m-auto">
-        {% for object in timeline_objects %}
-            <li{% cycle "" ' class="timeline-inverted"' %}>
-                <div class="timeline-badge {% if object.type == "start" %}bg-success{% elif object.type == "end" %}bg-danger{% else %}bg-info{% endif %}">
-                    <i class="icon-{{ object.model_name }}"></i>
-                </div>
-                <div class="card text-right">
-                    <div class="card-body">
-                        {{ object.event }}
-                        {% for detail in object.details %}
-                            <div><small>{{ detail }}</small></div>
-                        {% endfor %}
+        {% regroup timeline_objects by time as timeline_moments %}
+        
+        {% for moment_objects in timeline_moments %}
+            {% for object in moment_objects.list|dictsort:"type" %}
```

Comment:
This dictsort does the trick. However, the regroup is necessary to force sorting precedence on time, making this a bit more messy :-/

Reply:
I picked (1)